### ISSUE TITLE
Okular as flatpak from FlatHub can not utilized by Plank

### ISSUE THREAD

**Issue by Nokia808** [flathub/org.kde.okular#65]
Hi dear. This issue is exactly the same of that I already opened regarding PDF Mix Tool - kindly see the following:
https://gitlab.com/scarpetta/pdfmixtool/-/issues/38

As you see in above link, the developer of PDF Mix Tool fix it.

Please your kind corresponding fix. This issue limited to flatpak package from FlatHub.

**Comment by Nokia808** [comment]
Hi again. The developer of PDF Mix Tool said that:

> "The fix was to add this to the desktop file of the application:
> StartupWMClass=okular for Okular, if you want to report it upstream."
> 

Kindly to include this by updating the flatpak package of Okular on FlatHub.

**Comment by tsdgeos** [comment]
seems like a bug in plank to me, why would they require that if any other launch tool works just fine?

**Comment by Nokia808** [comment]
@tsdgeos 
I don't know because I'm not a programmer.

**Comment by eszlari** [comment]
Might be needed for GNOME Shell too:
https://github.com/obsproject/obs-studio/pull/2137

**Comment by TingPing** [comment]
A lot of tools assume desktop-id == startup-wm-class

**Comment by Nokia808** [comment]
Hi. This issue is fixed on version 1.10 !
When I launched Okular, there were 2 icons - see the following screenshots:

1st icon:
![icon-1](https://user-images.githubusercontent.com/25375330/80227238-e2d52280-863c-11ea-8c6b-9368e5f28bfd.png)

2nd icon:
![icon-2](https://user-images.githubusercontent.com/25375330/80227265-ebc5f400-863c-11ea-95d0-2b6ce79b5076.png)

When I tried 2nd icon (Goggle within blue) it did not work at all & disappeared from Plank just after closing Okular. But the 1st icon (regular icon of Okular) it worked when I tried it ! - see the following screenshot which showing 1st icon added successfully to Plank:

![plank](https://user-images.githubusercontent.com/25375330/80227711-8de5dc00-863d-11ea-9426-1c5882ade925.png)

I will colse this issue.
